src/mlpack/methods/nca/nca.hpp

```diff
@@ -73,9 +73,12 @@ class NCA
    * dataset.n_rows), that matrix will be used as the starting point for
    * optimization.
    *
+   * @tparam CallbackTypes Types of Callback functions.
    * @param output_matrix Covariance matrix of Mahalanobis distance.
+   * @param callbacks Callback Functions. 
```

Comment:
Same here about the ensmallen callbacks documentation link. :+1:

Reply:
Done :)